data/types/basal/suppressed_test.go

```diff
@@ -101,73 +102,77 @@ var _ = Describe("Target", func() {
 			},
 			Entry("parses object that is nil",
 				nil,
-				NewTestSuppressed(nil, nil, nil, nil, nil),
+				NewTestSuppressed(nil, nil, nil, nil, nil, nil),
 				[]*service.Error{}),
 			Entry("parses object that is empty",
 				&map[string]interface{}{},
-				NewTestSuppressed(nil, nil, nil, nil, nil),
+				NewTestSuppressed(nil, nil, nil, nil, nil, nil),
 				[]*service.Error{}),
 			Entry("parses object that has valid type",
 				&map[string]interface{}{"type": "basal"},
-				NewTestSuppressed("basal", nil, nil, nil, nil),
+				NewTestSuppressed("basal", nil, nil, nil, nil, nil),
 				[]*service.Error{}),
 			Entry("parses object that has invalid type",
 				&map[string]interface{}{"type": 0},
-				NewTestSuppressed(nil, nil, nil, nil, nil),
+				NewTestSuppressed(nil, nil, nil, nil, nil, nil),
 				[]*service.Error{
 					testData.ComposeError(service.ErrorTypeNotString(0), "/type", nil),
 				}),
 			Entry("parses object that has valid delivery type",
 				&map[string]interface{}{"deliveryType": "temp"},
-				NewTestSuppressed(nil, "temp", nil, nil, nil),
+				NewTestSuppressed(nil, "temp", nil, nil, nil, nil),
 				[]*service.Error{}),
 			Entry("parses object that has invalid delivery type",
 				&map[string]interface{}{"deliveryType": 0},
-				NewTestSuppressed(nil, nil, nil, nil, nil),
+				NewTestSuppressed(nil, nil, nil, nil, nil, nil),
 				[]*service.Error{
 					testData.ComposeError(service.ErrorTypeNotString(0), "/deliveryType", nil),
 				}),
 			Entry("parses object that has valid rate",
 				&map[string]interface{}{"rate": 2.0},
-				NewTestSuppressed(nil, nil, 2.0, nil, nil),
+				NewTestSuppressed(nil, nil, 2.0, nil, nil, nil),
 				[]*service.Error{}),
 			Entry("parses object that has invalid rate",
 				&map[string]interface{}{"rate": "invalid"},
-				NewTestSuppressed(nil, nil, nil, nil, nil),
+				NewTestSuppressed(nil, nil, nil, nil, nil, nil),
 				[]*service.Error{
 					testData.ComposeError(service.ErrorTypeNotFloat("invalid"), "/rate", nil),
 				}),
 			Entry("parses object that has valid schedule name",
 				&map[string]interface{}{"scheduleName": "Weekday"},
-				NewTestSuppressed(nil, nil, nil, "Weekday", nil),
+				NewTestSuppressed(nil, nil, nil, "Weekday", nil, nil),
 				[]*service.Error{}),
 			Entry("parses object that has invalid schedule name",
 				&map[string]interface{}{"scheduleName": 0},
-				NewTestSuppressed(nil, nil, nil, nil, nil),
+				NewTestSuppressed(nil, nil, nil, nil, nil, nil),
 				[]*service.Error{
 					testData.ComposeError(service.ErrorTypeNotString(0), "/scheduleName", nil),
 				}),
+			Entry("parses object that has valid annotations",
```

Comment:
as we have no validator at the mo is there any danger to just taking anything as being a valid annotation?

Reply:
Yes, technically there is. I haven't figured out a reasonable way to validate these "blobs". (The only way I can think of is to manually convert them into BSON or JSON and then calculate the string length, but that seems very "hacky", so I've left it off.)